config/env.ts

```diff
@@ -9,9 +9,11 @@ export const set_WORKSPACE_DIR = (dir: string) => process.env.GPTSCRIPT_WORKSPAC
 export const set_SCRIPTS_PATH = (dir: string) => process.env.SCRIPTS_PATH = dir;
 
 let gptscript: GPTScript | null = null;
+
 export function gpt() {
     if (!gptscript) {
         gptscript = new GPTScript()
-    };
+    }
+    ;
```

Comment:
Is this intentional or just rendering weird?

Reply:
Unintentional. I think formatter didn't like the existing `};` and decided to do this instead. Fixed.